### ISSUE TITLE
Inf. Stamina not working

### ISSUE THREAD

**Issue by fredso104** [bad1dea/NXCheats#56]
When sprinting, it seems like the cheat works but the stamina wheel is actually just invisible, the stamina still runs out and you need to wait for it to fill up again. When climbing, the stamina wheel does appear but it decreases like usual. Other cheat options work fine

**Comment by bad1dea** [comment]
I'm not sure what codes you are using, but the code works fine, and has been tested by others including myself..

https://youtu.be/maXhUgjIFbA

**Comment by fredso104** [comment]
Oh, do you have an idea as to why mine isn't working? Are there any variables that might effect it?

**Comment by fredso104** [comment]
I've just now noticed, there are some error codes upon game start up on ryujinx that weren't there before I applied the new updated cheats, is it possible for you to check them out and see if they are linked to your cheats?
00:00:02.091 |E| TamperMachine Compile: System.NullReferenceException: Object reference not set to an instance of an object.
   at Ryujinx.HLE.HOS.Tamper.InstructionHelper.GetCodeType(Byte[] instruction) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\InstructionHelper.cs:line 80
   at Ryujinx.HLE.HOS.Tamper.CodeEmitters.EndConditionalBlock.Emit(Byte[] instruction, CompilationContext context, IEnumerable`1 operationsElse) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\CodeEmitters\EndConditionalBlock.cs:line 44
   at Ryujinx.HLE.HOS.Tamper.CodeEmitters.EndConditionalBlock.Emit(Byte[] instruction, CompilationContext context) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\CodeEmitters\EndConditionalBlock.cs:line 20
   at Ryujinx.HLE.HOS.Tamper.AtmosphereCompiler.CompileImpl(String name, IEnumerable`1 rawInstructions) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\AtmosphereCompiler.cs:line 81
   at Ryujinx.HLE.HOS.Tamper.AtmosphereCompiler.Compile(String name, IEnumerable`1 rawInstructions) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\AtmosphereCompiler.cs:line 41
00:00:02.091 |E| TamperMachine Compile: There was a problem while compiling the Atmosphere cheat
00:00:02.092 |E| TamperMachine Compile: System.NullReferenceException: Object reference not set to an instance of an object.
   at Ryujinx.HLE.HOS.Tamper.InstructionHelper.GetCodeType(Byte[] instruction) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\InstructionHelper.cs:line 80
   at Ryujinx.HLE.HOS.Tamper.CodeEmitters.EndConditionalBlock.Emit(Byte[] instruction, CompilationContext context, IEnumerable`1 operationsElse) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\CodeEmitters\EndConditionalBlock.cs:line 44
   at Ryujinx.HLE.HOS.Tamper.CodeEmitters.EndConditionalBlock.Emit(Byte[] instruction, CompilationContext context) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\CodeEmitters\EndConditionalBlock.cs:line 20
   at Ryujinx.HLE.HOS.Tamper.AtmosphereCompiler.CompileImpl(String name, IEnumerable`1 rawInstructions) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\AtmosphereCompiler.cs:line 81
   at Ryujinx.HLE.HOS.Tamper.AtmosphereCompiler.Compile(String name, IEnumerable`1 rawInstructions) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Tamper\AtmosphereCompiler.cs:line 41
00:00:02.092 |E| TamperMachine Compile: There was a problem while compiling the Atmosphere cheat
